core/cli/src/commands/opt.rs

```diff
@@ -173,16 +173,26 @@ async fn status<C: Collection<SignerInterface = Signer<C>>>(
     let node_info = node_info.context("Node not found on state.")?;
 
     match node_info.participation {
-        Participation::True => println!("Your node is participating in the network."),
-        Participation::False => println!("Your node is not participating in the network."),
+        Participation::True => {
+            println!("Your node is participating in the network.");
+
```

Comment:
small nit but no need for newlines on these

Reply:
@ozwaldorf the empty lines? removed